rs/bitcoin/kyt/src/state.rs

```diff
@@ -122,3 +127,82 @@ impl Drop for FetchGuard {
         }
     }
 }
+
+#[derive(Clone, Debug)]
+pub struct Config {
+    pub network: Network,
+}
+
+impl Default for Config {
+    fn default() -> Self {
+        Self {
+            network: Network::Bitcoin,
+        }
+    }
+}
+
+impl From<InitArg> for Config {
+    fn from(arg: InitArg) -> Self {
+        let network = match arg.network {
+            BtcNetwork::Mainnet => Network::Bitcoin,
+            BtcNetwork::Testnet => Network::Testnet,
+            BtcNetwork::Regtest => Network::Regtest,
+        };
+        Self { network }
+    }
+}
+
+impl From<&Config> for InitArg {
+    fn from(arg: &Config) -> Self {
+        let network = match arg.network {
+            Network::Bitcoin => BtcNetwork::Mainnet,
+            Network::Testnet => BtcNetwork::Testnet,
+            Network::Regtest => BtcNetwork::Regtest,
+            _ => panic!("unsupported network type: Signet"),
+        };
+        Self { network }
+    }
+}
+
+impl Storable for Config {
+    fn to_bytes(&self) -> Cow<[u8]> {
+        let buf = encode_args((InitArg::from(self),)).expect("fail to encode config");
+        Cow::Owned(buf)
+    }
+
+    fn from_bytes(bytes: Cow<[u8]>) -> Self {
+        let (config,): (InitArg,) =
+            decode_args(bytes.as_ref()).expect("failed to decode config bytes");
+        config.into()
+    }
+
+    const BOUND: Bound = Bound::Unbounded;
+}
+
+type StableMemory = VirtualMemory<DefaultMemoryImpl>;
+
+const CONFIG_MEMORY_ID: MemoryId = MemoryId::new(0);
+
+// Configuration is stored in stable memory.
+thread_local! {
+    static MEMORY_MANAGER: RefCell<MemoryManager<DefaultMemoryImpl>> = RefCell::new(
+            MemoryManager::init(DefaultMemoryImpl::default())
+    );
+    static CONFIG: RefCell<Cell<Config, StableMemory>> = RefCell::new(
+        Cell::init(config_memory(), Config::default()).expect("failed to initialize stable cell for config")
```

Comment:
To avoid having an artificial `Config::default()` (because it should use the config from init args), you could use something like the approach in the LSO with the [ConfigState](https://github.com/dfinity/ic/blob/48f24e7e48590122130186e894d8096b12e9b772/rs/ethereum/ledger-suite-orchestrator/src/state/mod.rs#L465) enum which has 2 variants (uninitialized/initialized)

Reply:
Thanks for the suggestion! Pushed a commit to adopt the uninitialized/initialized approach.